src/applications/discharge-wizard/components/v2/resultsComponents/CarefulConsiderationStatement.jsx

```diff
@@ -151,10 +155,12 @@ const CarefulConsiderationStatement = ({ formResponses }) => {
                     If you want a DD214 for your period of honorable service for
                     other reasons, unrelated to applying for VA benefits, you
                     can request one.{' '}
-                    <Link to="/request-dd214" target="_blank">
-                      Find out how to request a DD214 for your period of
-                      honorable service (opens in a new tab)
-                    </Link>
+                    <va-link
```

Comment:
This link may need to open in a new tab since navigating away from this page would cause the user to lose some information (the results of the DUW). We can confirm with Fran maybe?

Reply:
@randimays This previously opened in a new tab but the behavior of this has it within the app/form flow (same title tag, same breadcrumbs, etc.) and so opening this in the same tab makes this confusing because everything else in the form flow opens in the same tab and then they can use the back button to get back. So I asked Chris specifically to change this and have it behave the same way as the rest of the form.

Reply:
Nice! Thanks.

Reply:
This component does rely on DUW responses. Previously opening in a new tab we needed to use local storage to save the data and retrieve it. Now it can use the redux store.

Reply:
Oh good callout. Thanks Chris!